admin/tools/docker/replication/container/tools/build.sh

```diff
@@ -34,7 +34,7 @@ docker run \
        -v /etc/group:/etc/group:ro \
        -v $HOME:$HOME \
        -v $PWD:$PWD \
-       qserv/replica:dev_conda \
+       qserv/replica:dev_conda_ceb6bb6 \
```

Comment:
Is linking to a specific version long term?

Reply:
I hope it's not! It's one of the consequences of the Conda migration of the LSST Stack. The hash in the container tag's name relates to some Qserv-specific overide in the LSST Stack environment:
{code}
LSST_SPLENV_REF=ceb6bb6
{code}
I'm using a specific hash to ensure I have a container based on the corresponding version of Stack.
I also hope this all will get much easier after migrating to the CMake based build system.

Reply:
And before that, as the LSST Stack will get more stable I may go back to the less elaborate naming convention for naming these containers:
```
qserv/replica:dev
```